articles/azure/asz-sco

```diff
@@ -0,0 +1,262 @@
+
+**Billing.** We provide you with monthly bills covering your monthly spend.
+
+## Customer service
+
+**Customer Success Manager (CSM).** All new customers of the service have access to a UKCloud Customer Success Manager, who will help you with your adoption of the UKCloud for Microsoft Azure service,  including finding relevant systems and using UKCloud tools and processes.
+
+**Cloud Architects.** UKCloud Cloud Architects support you during the design of solutions for the cloud platform. UKCloud Cloud Architects are ideally placed to help reconcile your requirements with the UKCloud platforms. We recommend engagement with a Cloud Architect when implementing complex solutions, such as those using HybridConnect or a Walled Garden.
+
+**Technical Account Managers (TAM).** You will be allocated with an assigned point of contact who will provide ongoing assistance with reporting and incident escalation.
+
+**Support.** After the initial on-boarding and design phase, you can utilise the standard UKCloud support entitlement, which is documented in the [*Customer Engagement Factsheet*](https://ukcloud.com/wp-content/uploads/2017/07/UKCloud-Factsheet-Customer-Care.pdf).
+
+## Customer responsibilities
+
+- You are responsible for defining which Azure Active Directory (AAD) you wish to link their UKCloud for Microsoft Azure service too. The customer can choose to link to their existing AAD, or ask UKCloud to create a new AAD solely for the use of this service.
+
+- You are responsible for assessing whether the UKCloud platform can support the various requirements of your application.
+
+- The control and management of access and responsibilities for end users including appropriate connectivity, security and accreditation if required.
+
+- If access is required over government secure networks (N3/HSCN, Janet, RLI or PSN), the customer is responsible for adhering to the relevant Code of Connection (CoCo) and for providing evidence of their CoCo to UKCloud upon request. UKCloud is unable to provide access to secure networks where such evidence has not been provided by the customer.
+
+- Management and administration of layers above the hypervisor (OS patching, application performance monitoring, user administration).
+
+- The customer is also responsible for ensuring only lawful data that supports the UK public sector is stored and processed by applications on this environment, and that they fully comply with the UKCloud Security Operating Procedures (SyOPs) and other information assurance requirements as specified in the UKCloud System Interconnect and Security Policy (SISP) and associated accreditation documentation sets.
```

Comment:
You are also responsible for ensuring only lawful data that supports the UK public sector is stored and processed by applications on this environment, and that you fully comply with the UKCloud Security Operating Procedures (SyOPs) and other information assurance requirements as specified in the UKCloud System Interconnect and Security Policy (SISP) and associated accreditation documentation sets.